code/src/components/RadioForm.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+
+const RadioForm = ({input, setInput}) => {
+    const rolesList = [
+        'Frontend',
+        'Backend',
+        'Fullstack',
+        'Cloud'
+    ]
+
+    return (
+        <div className="radio-form-div">
+            <div>
+                <h2>Interested in working with (select one)</h2>
+                {rolesList.map(role => (
+                <label key={role}>
+                <input 
+                    type="radio"
+                    value={role}
+                    name="roles"
+                    required
+                    onChange={event => setInput({...input, [event.target.name]: event.target.value})}
```

Comment:
It is nice to see how you implemented the spread syntax here. I am still trying to figure that out myself :)